webapp/pages/organization.spec.js

```diff
@@ -0,0 +1,38 @@
+import { mount } from '@vue/test-utils'
+import Organization from './organization.vue'
+import VueMeta from 'vue-meta'
+
+const localVue = global.localVue
+localVue.use(VueMeta, { keyName: 'head' })
+
+describe('organization.vue', () => {
+  let wrapper
+  let mocks
+
+  beforeEach(() => {
+    mocks = {
+      $t: (t) => t,
+    }
+  })
+
+  describe('mount', () => {
+    const Wrapper = () => {
+      return mount(Organization, {
+        mocks,
+        localVue,
+      })
+    }
+
+    beforeEach(() => {
+      wrapper = Wrapper()
+    })
+
+    it('renders', () => {
+      expect(wrapper.is('div')).toBe(true)
```

Comment:
toBeTruthy() instead of toBe(true)

Reply:
If it’s okay then I like to merge this PR and have it in my mind for the next time I touch it, because I needed half a day of restarting the test to get it through.